config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :pics do
```

Comment:
Good practice (again, according to review) to restrict to what routes you are actually using